src/features/create_layer/atoms/editableLayersListResource.ts

```diff
@@ -1,42 +1,15 @@
 import { createAsyncAtom } from '~utils/atoms/createAsyncAtom';
-import { createAtom } from '~utils/atoms/createPrimitives';
+import { appConfig } from '~core/app_config';
 import { apiClient } from '~core/apiClientInstance';
-import { currentApplicationAtom } from '~core/shared_state';
 import { EDITABLE_LAYERS_GROUP } from '~core/constants';
 import type { EditableLayers } from '../types';
 
-/**
- * This resource atom get editable user layers
- */
-const editableLayersListDependencyAtom = createAtom(
-  {
-    currentApplicationAtom,
-  },
-  (
-    { onChange },
-    state: {
-      appId: string | null;
-    } = {
-      appId: null,
-    },
-  ) => {
-    onChange('currentApplicationAtom', (currentApplication) => {
-      state = { appId: currentApplication };
-    });
-
-    return state;
-  },
-  'editableLayersListDependencyAtom',
-);
-
 export const editableLayersListResource = createAsyncAtom(
-  editableLayersListDependencyAtom,
-  async (params, abortController) => {
-    if (!params?.appId) return null;
-
+  null,
+  async (_, abortController) => {
     const responseData = await apiClient.post<EditableLayers[]>(
```

Comment:
should we also add it here? `if (!appConfig.id) return null;`

Reply:
appConfig.id cannot be empty

Reply:
so why to handle it here? 
https://github.com/konturio/disaster-ninja-fe/pull/402/files#diff-f926c1c66cd1d38ab061e5691b47a9059b0831317f6a5fe7ab9a7da93e8c9ac4R10

Reply:
fixed